consul/datadog_checks/consul/consul.py

```diff
@@ -103,11 +105,19 @@ def consul_request(self, instance, endpoint):
             else:
                 resp = requests.get(url, verify=cabundlefile, headers=headers)
 
+            resp.raise_for_status()
+
         except requests.exceptions.Timeout:
             self.log.exception('Consul request to {} timed out'.format(url))
+            self.service_check(self.CONSUL_CAN_CONNECT, self.CRITICAL, tags=service_check_tags)
+            raise
+        except Exception:
+            self.log.exception("Consul request to {} failed".format(url))
```

Comment:
Do we want to send the exception as the `message` part of the service check here?

Reply:
Oh yeah probably. I always forgot this param for service checks...